scripts/validator.js

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+'use strict';
+
+// Settings
+const settings = require('../settings/local');
+const master = Buffer.from('7e1b8e405919c1c8dcb07a677a2881b62517ecb7d136681851f514f22a806685', 'hex')
```

Comment:
Curious what this constant is.

Reply:
hex-encoded master private key for the playnet identity (public)